packages/ilmomasiina-backend/src/routes/auditlog/getAuditLogs.ts

```diff
@@ -0,0 +1,71 @@
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { Op, WhereOptions } from 'sequelize';
+
+import * as schema from '@tietokilta/ilmomasiina-models/src/schema';
+import { AuditLog } from '../../models/auditlog';
+import { stringifyDates } from '../utils';
+
+const MAX_LOGS = 100;
+
+export default async function getAuditLogItems(
+  request: FastifyRequest<{ Querystring: schema.AuditLoqQuery }>,
+  response: FastifyReply,
+): Promise<schema.AuditLogResponse> {
+  let where: WhereOptions<AuditLog>[] = [];
+  if (request.query.user) {
+    where = [
+      ...where,
+      { user: { [Op.like]: `%${request.query.user}%` } },
+    ];
+  }
+  if (request.query.ip) {
+    where = [
+      ...where,
+      { ipAddress: { [Op.like]: `%${request.query.ip}%` } },
+    ];
+  }
+  if (request.query.action) {
+    where = [
+      ...where,
+      { action: { [Op.in]: request.query.action } },
```

Comment:
(old issue) Since `action` comes from a query string, `in` makes little sense.
```suggestion
      { action: request.query.action },
```

Reply:
That's true when `action` is not an array.
See the older issue for better explanation.